src/test/java/org/codehaus/mojo/properties/PropertyResolverTest.java

```diff
@@ -81,6 +81,19 @@ public void multipleValuesAreResolved()
     }
 
     @Test
+    public void propertyIncludesAnotherPropertyMoreThanOnce()

```

Comment:
One of my concerns naming this method was choosing a name that would be as descriptive as possible.

I agree that `multipleOccurencesOfTheSameValueAreResolved` is probably grammatically more akin to the other tests.

Then I would suggest `multipleOccurencesOfTheSamePropertyWithinAnotherAreResolved` or`multipleOccurencesOfTheSamePlaceholderWithinAnotherAreResolved` as the terms "property" and "placeholder" seem to be used interchangeably within the file, and the thing being resolved to the value is the property/placeholder.

It's hard to name things right, and I do not really have a strong opinion in this case. I'd let the final word on this to @robertotru.bertotru